test/index-test.js

```diff
@@ -41,8 +44,46 @@ describe('renderReactWithAphrodite aphrodite css rendering', () => {
       global.window = window;
       global.document = window.document;
 
+      const hydrateMethod = sinon.spy(ReactDOM, 'hydrate');
+
+      renderReactWithAphrodite('AC', AphroditeComponent);
+
+      assert(hydrateMethod.calledOnce);
+
+      delete global.window;
+      delete global.document;
+
+      hydrateMethod.restore();
+
+      done();
+    });
+  });
+  it('does not blow up when calling renderReactWithAphrodite on the client (render method)', (done) => {
```

Comment:
there should be a blank line between the `it`s.

Reply:
complete to add blank